flink-connector-aws/flink-connector-dynamodb/src/main/java/org/apache/flink/connector/dynamodb/table/DynamoDbDynamicSink.java

```diff
@@ -62,7 +63,8 @@ protected DynamoDbDynamicSink(
             boolean failOnError,
             Properties dynamoDbClientProperties,
             DataType physicalDataType,
-            Set<String> overwriteByPartitionKeys) {
+            Set<String> overwriteByPartitionKeys,
+            Set<String> primaryKeys) {
```

Comment:
In DynamoDB nomenclature, this set of properties is a primaryKey (not primaryKeys). Let's use a proper name

Reply:
+1

Reply:
To be clear, you're just suggesting changing the variable name from `primaryKeys` to `primaryKey`, right?   And leaving its data type intact of `Set<String>`, since a Primary Key can consist of two fields - the Partition Key and the Sort Key?